docs/docs/installation.md

```diff
@@ -19,12 +19,14 @@ slug: /
 
     e. Using the official [Kubernetes manifest](https://github.com/oauth2-proxy/manifests) (Helm)
 
-    Prebuilt binaries can be validated by extracting the file and verifying it against the `sha256sum.txt` checksum file provided for each release starting with version `v3.0.0`.
+:::note
+Prebuilt binaries can be validated by extracting the file and verifying it against the `sha256sum.txt` checksum file provided for each release starting with version `v3.0.0`.
```

Comment:
I guess this could break the indentation of the list 🤔

Reply:
Fixed